### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted_YourFirstExtension_Zoom: Keyboard Focus is illogical while navigating inside the hamburger menu i.e. "Expand and Collapse menu" button at zoom 200%.

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2785]
“[Check out Accessibility Insights](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
Environment Details:
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : Narrator, NVDA2019.1.1
MAS Violated: MAS1.4.4, MAS 2.4.3

Repro Steps:
Navigate to the url: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension)
follow below steps:
Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
Navigate to "API" link.
From left navigation, select "Get Started".
Navigate to "Your First Extension" link.
Go to browser settings and increase the Zoom to 200%.
Go to the hamburger menu button and expand it,
Using keyboard observe the keyboard focus.

Actual:
Keyboard Focus is illogical while navigating inside the hamburger menu i.e. "Expand and Collapse menu" button

Expected:
Keyboard Focus must be logical and also must only iterate inside the hamburger button while navigating inside the hamburger menu i.e. "Expand and Collapse menu" button

Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

User Impact:
Keyboard only users will face difficulty while navigating through the hamburger menu button if the keyboard focus is illogical in the scenario.

MAS Reference:
MAS 1.4.4 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d})
MAS 2.4.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

Attachment for Reference:
[A11y_code.visualstudio.com_APIGetStarted_YourFirstExtension_Keyboard.zip](https://github.com/microsoft/vscode-docs/files/3320383/A11y_code.visualstudio.com_APIGetStarted_YourFirstExtension_Keyboard.zip)

**Comment by Mohini78** [comment]
Updating the bug as the previous bug :A11y_code.visualstudio.com_APIGetStarted_YourFirstExtension_Usability: No action feedback is provided when "Zoom in" or "Zoom out" button - is hit does not hold any blocking scenario and action feedback is not essential for screen reader users, low vision users or keyboard only users.

**Comment by gregvanl** [comment]
@Mohini78 I'm not sure what you mean by "illogical". Tab navigation seems to work as I would expect for the hamburger dropdown list, going up and down each item in the list.
